### DISCUSSION TITLE
Compilation error on "tasklist-theia" due to missing properties of the 'Container' class

### DISCUSSION THREAD

**Discussion by rhumbertgz** [eclipse-glsp/glsp#1097] [Q&A] [answered]
Hi,

Trying to run the `node-json-theia` I get this error build building it.

Any idea what could be the root of the problem? I get the same error locally and in the dev container.

```
src/browser/tasklist-diagram-configuration.ts:28:32 - error TS2345: Argument of type 'import("/workspaces/chameleon/node_modules/inversify/lib/container/container").Container' is not assignable to parameter of type 'import("/workspaces/chameleon/node_modules/@theia/core/node_modules/inversify/lib/container/container").Container'.
  Type 'Container' is missing the following properties from type 'Container': _activations, _deactivations, _moduleActivationStore, unloadAsync, and 31 more.

28         configureDiagramServer(container, GLSPTheiaDiagramServer);
                                  ~~~~~~~~~

src/browser/tasklist-diagram-configuration.ts:29:9 - error TS2322: Type 'import("/workspaces/chameleon/node_modules/inversify/lib/container/container").Container' is not assignable to type 'import("/workspaces/chameleon/node_modules/@theia/core/node_modules/inversify/lib/container/container").Container'.

29         return container;
           ~~~~~~~~~~~~~~~~~


Found 2 errors in the same file, starting at: src/browser/tasklist-diagram-configuration.ts:21
```

**Comment by planger** [comment, marked as answer]
Hi @rhumbertgz,

I'm sorry to hear that!
The [CI](https://ci.eclipse.org/glsp/job/eclipse-glsp/job/glsp-examples/job/master/) is green and has a [successful build yesterday](https://ci.eclipse.org/glsp/job/eclipse-glsp/job/glsp-examples/job/master/84/), and to retest, I did a fresh check-out of master and ran `cd project-templates/node-json-theia && yarn build && cd glsp-client && yarn start` and everything started without any build errors.

Which node version are you using?
> $ node -v
> v16.20.2

Which commands did you execute?

**Reply by rhumbertgz** [reply]
Hi @planger, after rebuild the container and the project the error was gone. Thanks again for your time 👍🏼
